### PR TITLE
Forward compatibility for new advanced buttons

### PR DESCRIPTION
**Pull request by NotMyFault**
https://github.com/jenkinsci/jenkins/pull/7173

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -44,7 +44,7 @@ branches['CI'] = {
 for (int i = 0; i < splits.size(); i++) {
   int index = i
   for (int j in [11]) {
-    for (String v in ['lts', 'latest']) {
+    for (String v in ['latest']) {
```
Comment (timja) [inline Jenkinsfile:47]:
```suggestion
    for (String v in ['lts', 'latest']) {
```

**Comment by janfaracik** [comment]
Looks like it's passing now 🙌

File: src/main/resources/ath-container/run.sh [hunk 2]
```diff
@@ -44,7 +44,7 @@ if [ ! -f $war ]; then
     case "$war" in
         "latest")
             war=jenkins-latest.war
-            url=https://updates.jenkins.io/latest/jenkins.war
+            url=https://repo.jenkins-ci.org/incrementals/org/jenkins-ci/main/jenkins-war/2.379-rc33144.0b_b_a_b_48a_744b/jenkins-war-2.379-rc33144.0b_b_a_b_48a_744b.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:47]:
```suggestion
            url=https://updates.jenkins.io/latest/jenkins.war
```

**Review by timja — APPROVED** [review]

File: src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java [hunk 3]
```diff
@@ -78,7 +78,7 @@ public static <T extends ToolInstallation> T addTool(Jenkins jenkins, Class<T> t
         final ConfigurablePageObject page = ensureConfigPage(jenkins);
 
         final String name = type.getAnnotation(ToolInstallationPageObject.class).name();
-        final Control expandButton = page.control(by.button(name + " installations..."));
+        final Control expandButton = page.control(by.button(name + " installations"));
```
Comment (jtnord) [inline src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java:81]: I am confused as to how is this (and other changes) compatible with the current Jenkins version given this should not match the existing button?

Comment (timja) [reply src/main/java/org/jenkinsci/test/acceptance/po/ToolInstallation.java:81]: The locator does it based off of contains string:
https://github.com/jenkinsci/acceptance-test-harness/blob/51712c7c3c63bbc09e7b2175cb5e38eecd638ccc/src/main/java/org/jenkinsci/test/acceptance/ByFactory.java#L170

**Pull request merged**